hilti/runtime/src/library.cc

```diff
@@ -118,14 +118,10 @@ hilti::rt::Result<Nothing> hilti::rt::Library::remove() const {
 }
 
 uint64_t hilti::rt::Library::currentScope() {
-    std::string scope;
-
     if ( _current_path )
-        scope = _current_path->native();
-    else
-        scope = fmt("<scope-%>" PRIu64, ++_scope_counter);
+        return std::hash<std::string>{}(_current_path->native());
 
-    return std::hash<std::string>()(scope);
+    return ++_scope_counter;
```

Comment:
Let's skip this commit, I really dislike returning a hash in one case and a meaningful value is another.